docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1c.md

```diff
@@ -0,0 +1,65 @@
+ 	2. the average: (each space coincident peak W) / total_area
+
+**Inputs:** 
+- **B_RMD**
+- **zone**
+- **zones_and_systems** - this is a dict of the existing expected system types from the function `get_zone_target_baseline_system`
+
+**Returns:**  
+- **result**: an enum - either YES or NO
+ 
+**Function Call:**
+- **get_baseline_system_types**
+- **get_zone_peak_internal_load**
+- **get_list_hvac_systems_associated_with_zone**
+- **get_hvac_zone_list_w_area**
+- **find_zone**
+- **get_zone_eflh**
+- **zones_with_computer_room_dict_x**
+
+## Logic:
+- set the result variable to "No" - only a positive test can give it a different value: `result = NO`
+
+- first check if the system type is one of the following: `eligible_primary_system_types = ["SYS-5","SYS-6","SYS-7","SYS-8"]`
+- check if the starting_system_type is in the list of eligible system types: `if zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] in eligible_primary_system_types:`
+	- create a list of zones that are on the same floor that have the same system type: `zones_on_same_system = []`
+	- create a variable to hold the sum total of the system area:	`system_total_area = 0`
+	- create a list of the EFLH's for each zone `eflh = []`
+	- create a variable to hold the sum total of all peak loads in the system: `system_total_peak_load = 0`
+	- loop through the zones in the zones_and_systems list: `for z in zones_and_systems:`
+		- make sure that we don't include the original zone: `if z != zone:`
+			- determine whether it has the same floor: `if z.floor_name == zone.floor_name:`
+				- determine whether it has the same system type: `if zones_and_systems[z]["EXPECTED_SYSTEM_TYPE"] == zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"]:`
+					- add the zone internal loads to the system_total_peaks: `a = get_zone_peak_internal_load(z)`
+					`system_total_peak += a["PEAK"]`
+					- get the zone eflh by using the get_zone_eflh function.  This returns eflh for the year, so we need to convert to weeks by dividing by 52.1428: `zone_eflh = get_zone_eflh(z)/52.1428`
+					- add the eflh to the eflh list.  We are multiplying the schedule by the zone area so that later we can divide the total by the total area.  Otherwise small zones with very different schedules have too great an impact
+					`eflh.append(zone_eflh * a["AREA"])`
+		- now calculate the average internal load: `avg_internal_load = system_total_peak / system_total_area`
+		- and calculate the averge eflh: `avg_eflh = sum(eflh)/len(eflh)`
+		- get the internal load of the reference zone: `internal_loads = get_zone_peak_internal_load(zone)`
+		- now do the rule checks:
+		- if the zone peak differs by more than 10 btu/hr/sf from the average, then it meets the exception:
+		`if(abs(internal_loads["PEAK"]/internal_loads["AREA"] - avg_internal_load) > 10):
+			- it meets the exception: `result = YES`
+		- OR if the eflh differs by more than 40, then the zone meets this exception:
+		`if(abs(get_zone_eflh(zone) - eflh) > 40):
```

Comment:
seems like you should be subtracting avg_eflh, not eflh

Reply:
yes, correct